doc/reference/reference_lua/datetime.rst

```diff
@@ -1,62 +1,964 @@
+                                If an empty table or no arguments are passed, the ``interval`` object with the default value ``0 seconds`` is created.
+
+    :return: interval_object
+    :rtype: cdata
+
+    ..  _interval-new-args:
+
+    **Possible input time units and parameters for datetime.interval.new()**
+
+    ..  container:: table
+
+        ..  list-table::
+            :widths: 20 50 20 10
+            :header-rows: 1
+
+            *   -   Name
+                -   Description
+                -   Type
+                -   Default
+
+            *   -   nsec (usec, msec)
+                -   Fractional part of the last second. You can specify either nanoseconds (``nsec``), or microseconds (``usec``), or milliseconds (``msec``).
+                    Specifying two of these units simultaneously or all three ones lead to an error.
+                -   number
+                -   0
+
+            *   -   sec
+                -   Seconds
+                -   number
+                -   0
+
+            *   -   min
+                -   Minutes
+                -   number
+                -   0
+
+            *   -   hour
+                -   Hours
+                -   number
+                -   0
+
+            *   -   day
+                -   Day number
+                -   number
+                -   0
+
+            *   -   week
+                -   Week number
+                -   number
+                -   0
+
+            *   -   month
+                -   Month number
+                -   number
+                -   0
+
+            *   -   year
+                -   Year
+                -   number
+                -   0
+
+            *   -   adjust
+                -   Defines how to round days in a month after an arithmetic operation.
+                -   string
+                -   'none'
+
+    ..  _interval-new-example:
+
+    **Examples:**
+
+    ..  code-block:: tarantoolsession
+
+        tarantool> datetime.interval.new()
+
+        ---
+        - 0 seconds
+        ...
+
+        tarantool> datetime.interval.new {
+                    month = 6,
+                    year = 1
+                    }
+        ---
+        - +1 years, 6 months
+        ...
+
+        tarantool> datetime.interval.new {
+                    day = -1
+                    }
+        ---
+        - -1 days
+        ...
+
+
+
+
+..  _datetime-module-api-reference-properties:
+
+Properties
+~~~~~~~~~~
+
+..  _datetime-tz:
+
+..  data:: TZ
+
+    Since: :doc:`2.11.0 </release/2.11.0>`
+
+    An array of timezone names and abbreviations.
+
+
+
+..  _datetime-module-api-reference-objects:
+
+Related objects
+~~~~~~~~~~~~~~~
+
+.. _datetime_obj:
+
+datetime_object
+***************
+
+..  class:: datetime_object
+
+    A ``datetime`` object.
+
+    ..  _datetime-add:
+
+    ..  method:: add( input[, { adjust } ] )
+
+        Modify an existing datetime object by adding values of the input argument.
```

Comment:
Please add a link to the section "Datetime and interval arithmetic" as it contains more details about arithmetic.

Reply:
Thanks!